fulcio.proto

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.fulcio.v1;
+
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/sigstore/fulcio/pkg/generated/protobuf";
+option java_package = "dev.sigstore.fulcio.v1";
+option java_multiple_files = true;
+option java_outer_classname = "FulcioProto";
+
+service CA {
+    /**
+     * Returns information about the current state of the transparency log
+     */
+    rpc CreateSigningCertificate(CreateSigningCertificateRequest) returns (SigningCertificate){
+        option (google.api.http) = {
+          post: "/api/v2/signingCert"
+          body: "*"
+        };
+    }
+    /**
+     * Returns the bundle of certificates that can be used to validate the signed certificate timestamp
```

Comment:
This trust bundle also can also validate signatures from issued code-signing certs right? Can we add that to the comment here if so? It feels like an important aspect of what the trust bundle can be used for

Reply:
Actually the trust bundle can't verify the SCT, it can only verify the code-signing certs. The CT log key doesn't come from Fulcio (and probably shouldn't, we don't want to couple the two).

Reply:
Right ok! That makes more sense. We should amend this comment entirely then!